src/main/java/uk/gov/companieshouse/ocr/api/image/extracttext/ExtractTextResultDTO.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.companieshouse.ocr.api.image.extracttext;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Results returned to a client on a successful API call { "extracted-text":
+ * "This is the extracted extracted from the document",
+ * "average-confidence-score": "96", "lowest-confidence-score": "45",
+ * "processing-time-ms": "27363" }
+ * 
+ */
+public class ExtractTextResultDTO {
+
+    @JsonProperty("extracted_text")
+    private String extractedText;
+
+    @JsonProperty("average-confidence-score")
```

Comment:
this and the others should be snake_case like extracted_text

Reply:
updated